codice/alliance
# CAL-351 Separated static analysis into two steps, formatted jenkinsfile, added Codecov check

#### What does this PR do?
The static analysis step is now quality analysis and security analysis steps. The quality checks have been moved after the deploy step and codecov was added as another parallel task. The jenkinsfile was also formatted.

#### Who is reviewing it? 
@emanns95 @oconnormi @LinkMJB 

#### Choose 2 committers to review/merge the PR.
@clockard @shaundmorris @rzwiefel 

#### How should this be tested?
Run a multibranch pipeline build with this jenkinsfile

Any background context you want to provide?

The quality analysis stage contains items that should inform the development team without failing the build. However the security analysis stage must still be passed for the build to deploy.

#### What are the relevant tickets?

[CAL-351](https://codice.atlassian.net/browse/CAL-351)
